### PR TITLE
dev/menu reorder

### PR DESCRIPTION
**Pull request by savedra1**
- better short menu options
- tui code refactor
- remove unused funcs

### CODE REVIEW

File: app/appBase.go [hunk 1]
```diff
@@ -309,3 +234,32 @@ func filterItemsByPinned(clipboardItems []config.ClipboardItem, isPinned bool) [
 
 	return filteredItems
 }
+
+func pinnedStyle() string {
+	var color string
```
Comment (ccoVeille) [inline app/appBase.go:239]: I would use this to remove the else


```go
func pinnedStyle() string { 
         pinChar := " " 
         config := config.GetTheme() 
  
         color := "#FF0000" 
         if config.UseCustom { 
                 color = config.PinIndicatorColor 
         } 
         return lipgloss.NewStyle().Foreground(lipgloss.Color(color)).SetString(pinChar).Render() 
 }
```

File: app/appBase.go [hunk 2]
```diff
@@ -279,9 +190,23 @@ func (m model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 
 	return m, tea.Batch(cmds...)
 }
+func (m model) View() string {
```
Comment (ccoVeille) [inline app/appBase.go:193]: I would use this to remove the else

```go
func (m model) View() string { 
         listView := m.list.View() 
         parts := strings.Split(listView, "\n") 
  
         var helpView string 
         if !m.showFullHelp {
                 helpView = "  " + m.list.Help.ShortHelpView(m.keys.ShortHelp()) 
         }

       // ...
```

File: app/appBase.go [hunk 3]
```diff
@@ -309,3 +234,32 @@ func filterItemsByPinned(clipboardItems []config.ClipboardItem, isPinned bool) [
...
+	if config.UseCustom {
+		color = config.PinIndicatorColor
+	} else {
+		color = "#FF0000"
+	}
+	return lipgloss.NewStyle().Foreground(lipgloss.Color(color)).SetString(pinChar).Render()
+}
+
+// This updates the TUI when an item is pinned/unpinned
+func (m *model) togglePinUpdate() {
```
Comment (ccoVeille) [inline app/appBase.go:252]: Use this

```go
func (m *model) togglePinUpdate() { 
         index := m.list.Index() 
         i, ok := m.list.SelectedItem().(item);
         if !ok {
                 return
         }

         i.description = fmt.Sprintf("Date copied: %s", i.timeStamp) 
         if !i.pinned { 
                         i.description = fmt.Sprintf("Date copied: %s %s", i.timeStamp, pinnedStyle())
          }
         i.pinned = !i.pinned
         m.list.SetItem(index, i) 
 }
```

File: app/appBase.go (hunk 1 above)
Comment (savedra1) [reply app/appBase.go]: Thank you as always :pray:  I'm still getting used to the recommended Go design patterns.

**Pull request merged**